_includes/content/ssh-add-keys-to-agent.md

```diff
@@ -0,0 +1,92 @@
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#mac-add-keys" data-group="mac">Mac</a></li>
+  <li><a data-toggle="tab" data-target="#win-add-keys" data-group="win">Windows</a></li>
+  <li><a data-toggle="tab" data-target="#linux-add-keys" data-group="linux">Linux</a></li>
+</ul>
+<div class="tab-content">
+<div id="mac-add-keys" class="tab-pane fade in active">
+<br>
+{% capture mac-content-add %}
+1.  Start the `ssh-agent` in the background.
+
+    ```none
+    eval "$(ssh-agent -s)"
+    Agent pid 59566
+    ```
+
+2.  On macOS Sierra 10.12.2 or newer, modify your
+`~/.ssh/config` file to automatically load keys into the `ssh-agent` and store
+passphrases in your keychain.
+
+    ```none
+    Host *
+     AddKeysToAgent yes
+     UseKeychain yes
+     IdentityFile ~/.ssh/id_rsa
+    ```
+
+3.  Add your SSH private key to the ssh-agent, using the default macOS `ssh-add` command.
+
+    ```none
```

Comment:
Keeping same as GitHub instructions. Not sure what you mean. Feel free to update the file with a new PR later?